test/lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -0,0 +1,118 @@
+describe('lib/rules/validate-jsdoc/require-description-complete-sentence', function () {
+    var checker = global.checker({
+        additionalRules: ['lib/rules/validate-jsdoc.js']
+    });
+
+    describe('not configured', function() {
+
+        it('should report with undefined', function() {
+            global.expect(function() {
+                checker.configure({requireDescriptionCompleteSentence: undefined});
+            }).to.throws(/accepted value/i);
+        });
+
+        it('should report with an object', function() {
+            global.expect(function() {
+                checker.configure({requireDescriptionCompleteSentence: {}});
+            }).to.throws(/accepted value/i);
+        });
+
+    });
+
+    describe('with true', function() {
+        checker.rules({requireDescriptionCompleteSentence: true});
+
+        checker.cases([
+            /* jshint ignore:start */
+            {
+                it: 'should not report common cases',
+                code: function() {
+                    function fun(p) {
+                    }
+
+                    /**
+                     * @param p
+                     */
+                    function fun(p) {
+                    }
+                }
+            }, {
+                it: 'should report missing period at end of description',
+                code: function () {
+                    /**
+                     * Some description
+                     * @param {number} p description without hyphen
+                     */
+                    function fun(p) {
+                    }
+                },
+                errors: {
+                    line: 2,
+                    column: 18
+                },
+                it: 'should report missing period at end of multi line description',
+                code: function () {
+                    /**
+                     * Some description
+                     * That takes up multiple lines
```

Comment:
I think that is a great idea!
(And it was actually easy to implement!)

I have choosen for it to complain at the upper case letter saying:
You started a new line with an upper case letter but previous line does not end with a period